sources/packages/backend/libs/sims-db/src/entities/student.model.ts

```diff
@@ -150,4 +151,14 @@ export class Student extends RecordDataModel {
     cascade: false,
   })
   applications?: Application[];
+
+  /**
+   * Legacy student profile.
+   * The student can potentially be associated with multiple legacy profiles,
+   * but only one is currently supported.
+   */
+  @OneToMany(() => SFASIndividual, (sfasIndividual) => sfasIndividual.student, {
+    nullable: true,
+  })
+  sfasIndividuals?: SFASIndividual[];
```

Comment:
Is it not suppose to be One to One. Currently there are multiple associations which is not an expected data but our application logic to pull the legacy data is based on single association expectation. In both directions we can join only the latest record but we can keep the model to to represent 1 to 1.

Reply:
As we talked, this represents the current DB structure and there is no clear definition from biz about how multiple associations will be handled. This is a simple approach to allow the retrieval of the most update record and, is rares moments, it may return multiples rows also.